packages/related-keyphrase-suggestions/src/components/KeyphrasesTable/index.js

```diff
@@ -198,9 +198,11 @@ export const KeyphrasesTable = ( { columnNames = [], data, renderButton, related
 					</div>
 				</Table.Header>
 
-				{ renderButton && <Table.Header className="yst-w-32">
-					<div className="yst-flex yst-justify-end yst-text-right rtl:yst-text-left">
-						{ __( "Add keyphrase", "wordpress-seo" ) }
+				{ renderButton && <Table.Header>
+					<div className="yst-flex yst-justify-end yst-w-full">
+						<div className="yst-text-end yst-w-20">
```

Comment:
I find it weird that here we "enforce" a small width which goes to multiline in default English.
But the Difficulty is enforced nowrap regardless of how long the language makes it.
I think this is asking for bugs in other languages in favor of a certain style in English.

Reply:
In the difficulty index column it is used to make sure the percentage sign is not is a separate line, and it is one word, so that makes sense to me.
In the design the width of the column for the "Add keyphrase" was smaller than what it should have been. It became wider to facilitate the success message without having a jump in the column width.

Reply:
After looking at different translations for "Add keyphrase", the only issue was with French, fixed that by increasing the width slightly.